src/pages/dashboardGroups/components/GroupsTableProduct/components/GroupProductTableColumns.tsx

```diff
@@ -222,23 +190,28 @@ function showRefernts(params: GridRenderCellParams, onRowClick: (partyGroup: Par
     <React.Fragment>
       {renderCell(
         params,
-        <Box sx={{ backgroundColor: '#F5F5F5', borderRadius: '4px' }}>
-          <Typography
-            sx={{
-              overflow: 'hidden',
-              textOverflow: 'ellipsis',
-              display: '-webkit-box',
-              WebkitLineClamp: 2,
-              WebkitBoxOrient: 'vertical' as const,
-              width: '100%',
-              color: params.row.status === 'SUSPENDED' ? '#9E9E9E' : undefined,
-              fontSize: '14px',
-              fontWeight: 600,
-              padding: '4px',
-            }}
-          >
-            {`${(params.row as PartyGroup).membersCount} referenti`}
-          </Typography>
+        <Box display="flex">
+          <Box>
+            <GroupIcon color={isGroupSuspended(params.row as PartyGroup) ? 'disabled' : 'action'} />
+          </Box>
+          <Box>
+            <Typography
+              sx={{
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                display: '-webkit-box',
+                WebkitLineClamp: 2,
+                WebkitBoxOrient: 'vertical' as const,
+                width: '100%',
+                color: params.row.status === 'SUSPENDED' ? '#9E9E9E' : undefined,
+                fontSize: '14px',
+                fontWeight: 600,
```

Comment:
is this fontWeight with fixed value overridable with mui-italia variable?

Reply:
Yes. Modified.